.github/workflows/reusable-docker.yml

```
@@ -90,24 +86,23 @@ jobs:
         with:
           driver-opts: 'image=moby/buildkit:v0.13.1'
 
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v4
+      - id: 'auth'
+        name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@v1'
```

actionlint (1.7.4)

the runner of "google-github-actions/auth@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)